src/ray/protobuf/gcs.proto

```diff
@@ -325,6 +325,10 @@ message ResourcesData {
   ResourceLoad resource_load_by_shape = 7;
   // Whether this node manager is requesting global GC.
   bool should_global_gc = 8;
+  // Changes in resources used by normal task.
+  // Key is resource name, value is resource change.
+  // If the task returns the resource, the value is negative.
+  map<string, double> normal_task_resources_changes = 9;
```

Comment:
Do we still need `resources_available `? Do we have it there because we are in transition now?

Reply:
Yeah, because the resources of actor have not been deducted in GCS, it needs to wait for this PR.